src/modules/DASDController.rb

```diff
@@ -360,7 +357,7 @@ def Summary
     def probe_or_mock_disks
       mock_filename = ENV["YAST2_S390_PROBE_DISK"]
       if mock_filename
-        YAML.load(File.read(mock_filename))
+        YAML.safe_load(File.read(mock_filename))
```

Comment:
that is only tricky change. As it is for testing probe, do you try it? Sometimes if we export whole ruby object, then safe_load breaks it.

Reply:
YAST2_S390_PROBE_DISK=test/data/probe_disk_dasd.yml rake run[dasd] works fine, but maybe it is safer to not change it by now, what do you think?

Reply:
if it works, lets keep change. It is safer and looks like we do not export any non trivial objects.